Nextjs build - Module not found: Can't resolve .../include-defaults.js

**Description:**
When trying to do a build `npm run build`, it fails with several instances of not being able to find that file (even though it's exactly where it says it should be).


**Actual result:**
```info  - Creating an optimized production build
Failed to compile.

./node_modules/next/dist/client/add-base-path.js
Module not found: Can't resolve '..\..\..\@module-federation\nextjs-mf\src\include-defaults.js' in 'C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\next\dist\client'
Did you mean './..\..\..\@module-federation\nextjs-mf\src\include-defaults.js'?
Requests that should resolve in the current directory need to start with './'.
Requests that start with a name are treated as module requests and resolve within module directories (node_modules, C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe).
If changing the source code is not an option there is also a resolve options called 'preferRelative' which tries to resolve these kind of requests in the current directory too.

Import trace for requested module:
./node_modules/next/dist/shared/lib/router/router.js
./node_modules/next/dist/client/router.js

./node_modules/next/dist/client/add-locale.js
Module not found: Can't resolve '..\..\..\@module-federation\nextjs-mf\src\include-defaults.js' in 'C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\next\dist\client'
...
```

Also the same error for these files:
./node_modules/next/dist/client/detect-domain-locale.js
./node_modules/next/dist/client/has-base-path.js
./node_modules/next/dist/client/head-manager.js

**Expected result:** The build works.

**nextConfig:**
```
/** @type {import('next').NextConfig} */
import NextFederationPlugin from '@module-federation/nextjs-mf'
import { i18n } from '@telus/next-locale-middleware/config.js'

const nextConfig = {
  webpack: (config, options) => {
    if (!options.isServer) {
      config.plugins.push(
        new NextFederationPlugin({
          name: 'mfeInventory',
          remotes: {},
          filename: 'static/chunks/remoteEntry.js',
          exposes: {
            './Home': './src/components/Home/index.js'
          },
          shared: {
            react: {
              requiredVersion: false,
              singleton: true
            }
          },
          extraOptions: {
            skipSharingNextInternals: true
          }
        })
      )
    }
    return config
  },
  reactStrictMode: true,
  i18n
}

export default nextConfig

```

**package versions:**
"@module-federation/nextjs-mf": "^5.9.11",
"next": "^12.2.0",
"typescript": "4.7.4"

When using `yarn` instead of `npm`, I didn't get the above. Instead, I got `Error: Cannot find module 'webpack/lib/container/options'`. This was resolved adding webpack to me dev dependencies `yarn add -D webpack@5.61.0`. I thought webpack wasn't needed, as per the examples. Anyhoot... npm sucks I guess.

Also, I get this message during my build...
```
info  - Creating an optimized production build ...<w> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Resolving 'next/dist/compiled/webpack/webpack-lib' in C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\node\src\plugins for build dependencies doesn't lead to expected result 'C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\next\dist\compiled\webpack\webpack-lib.js', but to 'C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\node\node_modules\next\dist\compiled\webpack\webpack-lib.js' instead. Resolving dependencies are ignored for this path.
<w>  at unknown 4 next/dist/compiled/webpack/webpack-lib
<w>  at file dependencies C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\node\src\plugins\LoadFileChunkLoadingRuntimeModule.js
<w>  at file C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\node\src\plugins\LoadFileChunkLoadingRuntimeModule.js
<w>  at file dependencies C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\node\src\plugins\CommonJsChunkLoadingPlugin.js
<w>  at file C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\node\src\plugins\CommonJsChunkLoadingPlugin.js
<w>  at file dependencies C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\node\src\plugins\StreamingTargetPlugin.js
<w>  at file C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\node\src\plugins\StreamingTargetPlugin.js
<w>  at file dependencies C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\node\src\plugins\UniversalFederationPlugin.js
<w>  at file C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\node\src\plugins\UniversalFederationPlugin.js
<w>  at file dependencies C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\node\src\index.js
<w>  at file C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\node\src\index.js
<w>  at file dependencies C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\nextjs-mf\src\plugins\ChildFederationPlugin.js
<w>  at file C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\nextjs-mf\src\plugins\ChildFederationPlugin.js
<w>  at file dependencies C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\nextjs-mf\src\plugins\NextFederationPlugin.js
<w>  at file C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\nextjs-mf\src\plugins\NextFederationPlugin.js
<w>  at file dependencies C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\nextjs-mf\src\index.js
<w>  at file C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\node_modules\@module-federation\nextjs-mf\src\index.js
<w>  at resolve esm file @module-federation/nextjs-mf
<w>  at file dependencies C:\node-projects\telus\tps\cdo-tps-product-inventory-mfe\next.config.mjs
```

--- a/packages/nextjs-mf/src/loaders/patchDefaultSharedLoader.ts
+++ b/packages/nextjs-mf/src/loaders/patchDefaultSharedLoader.ts
@@ -25,7 +25,7 @@ export default function patchDefaultSharedLoader(
 
   return [
     '',
-    `require(${JSON.stringify(pathIncludeDefaults)});`,
+    `require(${JSON.stringify('./' + pathIncludeDefaults)});`,
     content
   ].join("\n")
 

